packages/types/responses.ts

```diff
@@ -217,10 +253,18 @@ export const ZResponseFilterCriteria = z.object({
 
   meta: z
     .record(
-      z.object({
-        op: z.enum(["equals", "notEquals"]),
-        value: z.union([z.string(), z.number()]),
-      })
+      z.union([
+        z.object({
+          op: z.enum(["equals", "notEquals"]),
+          value: z.union([z.string(), z.number()]),
+        }),
```

Comment:
Please replace this object with 
`ZResponseFilterCriteriaDataEquals,
ZResponseFilterCriteriaDataNotEquals,`